.php_cs.dist

```diff
@@ -0,0 +1,173 @@
+<?php
+$finder = array_reduce(
+    [
+        __DIR__ . '/src/',
+        __DIR__ . '/bin/',
+        __DIR__ . '/tests/'
+    ],
+    function (PhpCsFixer\Finder $finder, $dir) {
+        return $finder->in($dir);
+    },
+    PhpCsFixer\Finder::create()->ignoreUnreadableDirs()
+)->notPath('/stubs/i')->notName('CallMap.php');
+$rules = [
+    '@PSR2' => true,
+    'array_syntax' => [
+        'syntax' => 'short',
+    ],
+    'binary_operator_spaces' => true,
+    'blank_line_after_opening_tag' => false,
+    'blank_line_before_return' => true,
+    'cast_spaces' => false,
+    'class_keyword_remove' => false,
+    'combine_consecutive_unsets' => true,
+    'concat_space' => [
+        'spacing' => 'one',
+    ],
+    /* // to be removed/altered if vimeo/psalm ever goes php:^7.0
+    'declare_equal_normalize' => [
+        'space' => 'none',
+    ], // to be removed/altered if vimeo/psalm ever goes php:^7.0
+    'declare_strict_types' => true,
+    */
+    'function_to_constant' => true,
+    'include' => false,
+    'linebreak_after_opening_tag' => true,
+    'lowercase_cast' => true,
+    'magic_constant_casing' => true,
+    'mb_str_functions' => false,
+    'method_separation' => true,
+    'native_function_casing' => true,
+    // to discuss
+    'native_function_invocation' => false,
+    'new_with_braces' => false,
+    'no_alias_functions' => true,
+    'no_blank_lines_after_class_opening' => true,
+    'no_blank_lines_after_phpdoc' => true,
+    'no_blank_lines_before_namespace' => true,
+    'no_empty_comment' => true,
+    'no_empty_phpdoc' => true,
+    'no_empty_statement' => true,
+    'no_extra_consecutive_blank_lines' => true,
+    'no_leading_import_slash' => true,
+    'no_leading_namespace_whitespace' => true,
+    'no_mixed_echo_print' => true,
+    'no_multiline_whitespace_around_double_arrow' => true,
+    'no_multiline_whitespace_before_semicolons' => true,
+    'no_php4_constructor' => true,
+    'no_short_bool_cast' => true,
+    'no_short_echo_tag' => true,
+    'no_singleline_whitespace_before_semicolons' => true,
+    'no_spaces_around_offset' => true,
+    'no_trailing_comma_in_list_call' => true,
+    'no_trailing_comma_in_singleline_array' => true,
+    /* // to discuss
+    'no_unneeded_control_parentheses' => true,
+    */
+    'no_unreachable_default_argument_value' => true,
+    'no_unused_imports' => true,
+    /* // to discuss
+    'no_useless_else' => true,
+    */
+    'no_useless_return' => true,
+    'no_whitespace_before_comma_in_array' => true,
+    'no_whitespace_in_blank_line' => true,
+    'non_printable_character' => true,
+    'normalize_index_brace' => true,
+    /* // to discuss
+    'ordered_class_elements' => true,
```

Comment:
it could be pain on the beginning when it was introduced, but later it easier to maintain repo/PRs, as you have less conflicts (without this rule, ppl usually add new imports at end of them, which leads to conflicts)